ssh/tailssh/tailssh.go

```diff
@@ -1672,7 +1548,7 @@ func (ss *sshSession) startNewRecording() (_ *recording, err error) {
 	} else {
 		var errChan <-chan error
 		var attempts []*tailcfg.SSHRecordingAttempt
-		rec.out, attempts, errChan, err = ss.connectToRecorder(ctx, recorders)
+		rec.out, attempts, errChan, err = ConnectToRecorder(ctx, recorders, ss.conn.srv.lb.Dialer())
```

Comment:
then you can pass in `ss.conn.srv.lb.Dialer().UserDial` here

Reply:
> ss.conn.srv.lb.Dialer().UserDial

But then I need to expose the whole LocalBackend of tsnet.Server? Isn't that worse than just exposing the dialer?

Reply:
tsnet.Server has a Dial func already that way you won't have to export anything new